fast_api_als/routers/submit_lead.py

```diff
@@ -38,7 +38,8 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
     
     if not db_helper_session.verify_api_key(apikey):
         # throw proper fastpi.HTTPException
-        pass
+        logging.error(f'Invalid API key {apikey}')
```

Comment:
check

Reply:
I fixed it in final commit

Reply:
Should I create a new branch and do all changes again